<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="new_urdf">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="0.0352403998115288 -0.00193856893882958 0.244321687293213"
        rpy="0 0 0" />
      <mass
        value="1.79096310966132" />
      <inertia
        ixx="0.00405760471142835"
        ixy="2.12937141956187E-06"
        ixz="3.3130303814979E-05"
        iyy="0.00267025475463101"
        iyz="-3.27699598315326E-07"
        izz="0.00229351602428316" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.752941176470588 0.752941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="link1">
    <inertial>
      <origin
        xyz="0.00498327717775757 0.0193666674481396 -0.0208319699436081"
        rpy="0 0 0" />
      <mass
        value="0.295860371929211" />
      <inertia
        ixx="0.000180259743700536"
        ixy="6.44638953623284E-09"
        ixz="9.63548170761076E-06"
        iyy="0.000222094785214239"
        iyz="1.23138835073157E-07"
        izz="0.000156316536855164" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot/meshes/link1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.752941176470588 0.752941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot/meshes/link1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint1"
    type="revolute">
    <origin
      xyz="0.00995 -0.17755 0.25365"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="link1" />
    <axis
      xyz="0 0 -1" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="200"
      velocity="3" />
  </joint>
  <link
    name="link2">
    <inertial>
      <origin
        xyz="-0.000340321831924395 -0.00038973697772049 -0.0520549751495829"
        rpy="0 0 0" />
      <mass
        value="0.149963427564102" />
      <inertia
        ixx="0.000183592447078298"
        ixy="8.28241022079025E-12"
        ixz="5.04253434467308E-09"
        iyy="0.000209336531842732"
        iyz="1.86610556551535E-06"
        izz="3.28235947222695E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot/meshes/link2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.752941176470588 0.752941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot/meshes/link2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint2"
    type="revolute">
    <origin
      xyz="0 -0.0154 -0.0559"
      rpy="0 0 0" />
    <parent
      link="link1" />
    <child
      link="link2" />
    <axis
      xyz="0 1 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="200"
      velocity="3" />
  </joint>
  <link
    name="link3">
    <inertial>
      <origin
        xyz="1.2276E-07 -0.0016186 -0.057031"
        rpy="0 0 0" />
      <mass
        value="0.13616" />
      <inertia
        ixx="7.4786E-05"
        ixy="1.4369E-10"
        ixz="9.1386E-10"
        iyy="9.2185E-05"
        iyz="2.2914E-07"
        izz="2.2865E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot/meshes/link3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot/meshes/link3.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint3"
    type="revolute">
    <origin
      xyz="0 0.017 -0.09955"
      rpy="0 0 0" />
    <parent
      link="link2" />
    <child
      link="link3" />
    <axis
      xyz="0 -1 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="200"
      velocity="3" />
  </joint>
  <link
    name="link4">
    <inertial>
      <origin
        xyz="0.00419043319213264 -0.0191704255225077 -0.020444114512171"
        rpy="0 0 0" />
      <mass
        value="0.28593190191178" />
      <inertia
        ixx="0.000177533812769103"
        ixy="1.36274634729717E-07"
        ixz="1.21200199346044E-05"
        iyy="0.000215348566469664"
        iyz="-2.26454170217403E-08"
        izz="0.000151424209768655" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot/meshes/link4.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.752941176470588 0.752941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot/meshes/link4.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint4"
    type="revolute">
    <origin
      xyz="0.00995 0.17755 0.25365"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="link4" />
    <axis
      xyz="0 0 -1" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="200"
      velocity="3" />
  </joint>
  <link
    name="link5">
    <inertial>
      <origin
        xyz="0.000340321831860155 0.000389736977768645 -0.0520549751487307"
        rpy="0 0 0" />
      <mass
        value="0.149963427561676" />
      <inertia
        ixx="0.000183592447077726"
        ixy="8.28239172928839E-12"
        ixz="-5.04253451928812E-09"
        iyy="0.000209336531842107"
        iyz="-1.86610556545494E-06"
        izz="3.28235947222027E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot/meshes/link5.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.752941176470588 0.752941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot/meshes/link5.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint5"
    type="revolute">
    <origin
      xyz="0 0.0154 -0.0559"
      rpy="0 0 0" />
    <parent
      link="link4" />
    <child
      link="link5" />
    <axis
      xyz="0 1 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="200"
      velocity="3" />
  </joint>
  <link
    name="link6">
    <inertial>
      <origin
        xyz="-1.33821760395225E-07 0.00161858205668938 -0.0570305226452938"
        rpy="0 0 0" />
      <mass
        value="0.136158125277113" />
      <inertia
        ixx="7.4786412779994E-05"
        ixy="1.42932928820051E-10"
        ixz="-1.01332567226857E-09"
        iyy="9.21854584482063E-05"
        iyz="-2.29245663693843E-07"
        izz="2.28646686168041E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot/meshes/link6.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.752941176470588 0.752941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot/meshes/link6.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint6"
    type="revolute">
    <origin
      xyz="0 -0.017 -0.09955"
      rpy="0 0 0" />
    <parent
      link="link5" />
    <child
      link="link6" />
    <axis
      xyz="0 -1 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="200"
      velocity="3" />
  </joint>
</robot>
